apigee_edge.module

```diff
@@ -38,6 +38,7 @@ use Drupal\apigee_edge\Form\DeveloperSettingsForm;
 use Drupal\apigee_edge\JobExecutor;
 use Drupal\apigee_edge\Plugin\Field\FieldType\ApigeeEdgeDeveloperIdFieldItem;
 use Drupal\apigee_edge\Plugin\Validation\Constraint\DeveloperEmailUniqueValidator;
+use Drupal\apigee_edge\UserRemovalHandler;
```

Comment:
Maybe place this class and the interface under the `Drupal\apigee_edge\User` namespace

Reply:
Refactored